CHANGES.md

```diff
@@ -1,3 +1,10 @@
+2024-11-11, Version 4.1.6
+=========================
+
+Placeholder for package maintainer
+
+ * feat: Override default strong-soap behaviour when mapping undefined element values to self closing tags (<tag/> instead of <tag xsi:nil=true></tag>. Option is ignoreAttributesUndefined, to enable option set to false. (Jamie Nicholson)
+
```

Comment:
No need to manually update this file.

It is auto-generated by `strong-tools` as part of our [publishing workflow](https://github.com/loopbackio/loopback-governance/blob/89a7104d9582cd5718242717a61c703fe41e7c2e/docs/publish-modules/publish-other-modules.md).